### PR TITLE
319 fix comments in 33

### LINKED ISSUE #319: Fix comments in 3.3

### PR DESCRIPTION
**Pull request by CasperStaahl**
closes #319

### CODE REVIEW

File: report/src/main.tex
Comment (sebastianbot6969) [inline report/src/main.tex:1]: the comma should not stand alone

File: report/src/sections/04-theory/02-abstract-semantics.tex [hunk 1]
```diff
@@ -266,16 +269,12 @@ \subsubsection{Abstract Semantics of SQL statements}
...
-    &\mathsf{List} \Bigl( \pi_{\mathbf{v}_t} \bigl(v_d \downarrow_{\true}^{(\env{\ab{d}}, \env{\absvars})} b \cup v_d \downarrow_{\unknown}^{(\env{\ab{d}}, \env{\absvars})} b \bigr) \Bigr) \into \lookupcl(v_a) ]\biggr)
-\end{split}\label{eq:equation123}
-\end{equation}
+\begin{multline}
+    \abssqlsem{\langle select(v_a, v_d, \mathbf{v}_t), b \rangle}(\rho_{\ab{d}}, \rho_{\ab{a}}) = \left(\rho_{\ab{d}}, \rho_{\ab{a}} [ v_a \mapsto \ab{t} ] \right)\\
+    \text{where } \ab{t} = \left( \mathsf{Col} \; \left( \pi_{\mathbf{v}_t} \left(v_d \downarrow_{\true}^{(\env{\ab{d}}, \env{\absvars})} b \cup v_d \downarrow_{\unknown}^{(\env{\ab{d}}, \env{\absvars})} b \right) \right) \right) \into \lookupcl(v_a)
+\end{multline}\label{eq:equation123}
 
 For \textit{insert}, we evaluate each expression in $\mathbf{e}$, and take the product of them as the set of abstract tuples to be inserted.
 We take the product as each expression might evaluate to a $\mathsf{list}$ type.
```
Comment (sebastianbot6969) [inline report/src/sections/04-theory/02-abstract-semantics.tex:280]: Shouldn't this be col?

File: report/src/main.tex
Comment (CasperStaahl) [reply report/src/main.tex:1]: fixed in newest commit

File: report/src/sections/04-theory/02-abstract-semantics.tex (hunk 1 above)
Comment (CasperStaahl) [reply report/src/sections/04-theory/02-abstract-semantics.tex]: Fixed in newest commit.

**Review by Actam55 — APPROVED** [review]

**Pull request merged**